alacritty.yml

```diff
@@ -480,62 +481,118 @@
 # - `action`: Execute a predefined action
 #
 #   - ToggleViMode
+#       Enable or Disable Using Vi key bindings
 #   - SearchForward
+#       Start a forward buffer search
 #   - SearchBackward
+#       Start a backward buffer search
 #   - Copy
+#       Copy the selection to the clipboard
 #   - Paste
+#       Paste from the system clipboard
 #   - PasteSelection
+#       Paste from the selection buffer
 #   - IncreaseFontSize
+#       Increase font size
 #   - DecreaseFontSize
+#       Decrease font size
 #   - ResetFontSize
+#       Reset font size to config value
 #   - ScrollPageUp
+#       Scroll Exactly One Page Up
 #   - ScrollPageDown
+#       Scroll Exactly One Page Down
 #   - ScrollHalfPageUp
+#       Scroll a half page up
 #   - ScrollHalfPageDown
+#       Scroll a half page down
 #   - ScrollLineUp
+#       Scroll one line up
 #   - ScrollLineDown
+#       Scroll one line down
 #   - ScrollToTop
+#       Scroll all the way to the top
 #   - ScrollToBottom
+#       Scroll all the way to the bottom
 #   - ClearHistory
+#       Clear the display buffer(s) to remove history
 #   - Hide
+#       Hide the Alacritty window
 #   - Minimize
+#       Minimize the Alacritty window
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
+#       Toggle full screen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty
 #   - ClearLogNotice
+#       Clear warning and error notices
 #   - ClearSelection
+#       Clear active selection
 #   - ReceiveChar
+#       Allow receiving character input
 #   - None
+#       No action
 #
 #   (`mode: Vi` only):
 #   - Open
+#       Attempt to open the element under the cursor, like a URL. See also url.launcher.
 #   - Up
+#       Move one character up
```

Comment:
This applies to all vi motions, but I'm going to put it just here to make the review a little less "spammy" than it already is.

I think we should clarify what is moved here. Just stating `Move one character up` might also just mean that the viewport is moved.

I'd suggest stating that the `vi mode cursor` is moved in all these motion documentations. In this case specifically, I'd suggest something like this:

```suggestion
#   - Up
#       Move the vi mode cursor up by one line
```

I think it makes sense to replace "one character" with "one line", since the idea of moving up by "one character" is a bit strange in a line based environment. For left/right motion it does make sense though (since a character might span over multiple columns).